taskManager.js

```diff
@@ -0,0 +1,93 @@
+const fs = require('fs');
+
+const TASKS_FILE = 'tasks.json'; // File where tasks are stored
```

Comment:
Use camel case naming

Reply:
A common convention is to use all-uppercase letters fo const